### PR TITLE
Add sequencing parameters to runs

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

File: core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/SequencingParameters.java [hunk 1]
```diff
@@ -2,39 +2,50 @@
...
-  public Date getLastUpdated();
+  public void setName(String string);
 
-  public void setLastUpdated(Date lastUpdated);
+  public void setPlatform(Platform platform);
+
+  public void setUpdatedBy(User updatedBy);
+
+  void setXPath(String xpath);
 }
```
Comment (callunity) [inline core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/SequencingParameters.java]: The re-alphabetizing of getters and setters makes this somewhat confusing, but it looks like everything is in (a new) order.

**Comment by callunity** [comment]
:+1:

**Comment by tonydebat** [comment]
:+1:

**Comment by apmasell** [comment]
There autowiring badness, but it doesn't break, so I'm merging.

**Pull request merged**